### PR TITLE
Removed make install & make uninstall target

### PR DESCRIPTION
**Pull request by Ygnas**
# Issue link

[RHOAIENG-5347](https://issues.redhat.com/browse/RHOAIENG-5347)

# What changes have been made
Removed make targets 

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [x] Testing is not required for this change

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,10 +34,9 @@ For building and pushing a new version of the operator image:
  - `make image-build -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
  - `make image-push -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
 
-For deploying onto a cluster:
+For running from your host:
  - First, either set `KUBECONFIG` or ensure you are logged into a cluster in your environment
- - `make install`
- - `make deploy -e IMG=<image-repo/image-name>`
+ - `make run`
```
Comment (sutaakar) [inline CONTRIBUTING.md:39]: AFAIK you need to provide namespace:
`NAMESPACE=default make run`

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Requirements:
   When you have a version of the GNU sed installed on a macOS you may specify the binary using
   ```bash
   # brew install gnu-sed
-  make install -e SED=/usr/local/bin/gsed
+  NAMESPACE=default make run -e SED=/usr/local/bin/gsed
```
Comment (zdtsw) [inline README.md:25]: why should this "make install" eventually become "make run" ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,10 +34,9 @@ For building and pushing a new version of the operator image:
  - `make image-build -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
  - `make image-push -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
 
-For deploying onto a cluster:
+For running from your host:
  - First, either set `KUBECONFIG` or ensure you are logged into a cluster in your environment
- - `make install`
- - `make deploy -e IMG=<image-repo/image-name>`
+ - `NAMESPACE=default make run`
```
Comment (zdtsw) [inline CONTRIBUTING.md:39]: why should use "default" ns here, and "make install deploy" becomes "go run"?

Comment (Ygnas) [reply CONTRIBUTING.md:39]: You are right here.

File: README.md (hunk 2 above)
Comment (Ygnas) [reply README.md:25]: Make install does not work at the moment, so people following the readme with get an error. Is changing it to make run bad here?

Comment (sutaakar) [reply README.md:25]: I would say that previous instructions were wrong, as `make install` just installed CRDs, however it didn't start the operator itself.
`make run` will start the operator now, connecting to the cluster from Kubeconfig.

Comment (sutaakar) [reply README.md:25]: @Ygnas Can you check if `-e SED=/usr/local/bin/gsed` is still needed. AFAIK there is no string replacement being done as part of `make run`

Comment (Ygnas) [reply README.md:25]: @sutaakar Yes `make run` does not need sed to run and it works fine. So would leaving only a note in this place fine?

Comment (zdtsw) [reply README.md:25]: if you want use "make run" here, just remove the "sed" part

Btw: @Fiona-Waters  + @dimakis looks like this PR: 
 https://github.com/project-codeflare/codeflare-operator/commit/4b863f79d4b704c511ea404e27e3ec65d66b5a0f removed the support of gsed on MacOS, was that a mistake?   FYI @tedhtchang

Comment (sutaakar) [reply README.md:25]: @Ygnas thinking, is SED used in any other command? If not then you can remove the note too.

Comment (sutaakar) [reply README.md:25]: ok, so just remove the SED from command

**Review by sutaakar — COMMENTED** [review]
/lgtm

File: README.md (hunk 2 above)
Comment (Fiona-Waters) [reply README.md]: > if you want use "make run" here, just remove the "sed" part
> 
> Btw: @Fiona-Waters + @dimakis looks like this PR: [4b863f7](https://github.com/project-codeflare/codeflare-operator/commit/4b863f79d4b704c511ea404e27e3ec65d66b5a0f) removed the support of gsed on MacOS, was that a mistake? FYI @tedhtchang

@zdtsw This was completed quickly just before a release as far as I remember. It was updated very soon afterwards in this [commit](https://github.com/project-codeflare/codeflare-operator/commit/c7954a9bd6b19f5978bcf318ebb7d839ad4b1b99) - a lot of this has been removed now with the removal of mcad. Hope I am understanding correctly.

**Comment by dgrove-oss** [comment]
We're going to be adding CRDs back again next week (when we bring in the AppWrapper operator).

**Comment by Ygnas** [comment]
> We're going to be adding CRDs back again next week (when we bring in the AppWrapper controller).

Closing because of this.

**Pull request closed without merging**